lib/sitemap-item.js

```diff
@@ -217,7 +217,7 @@ class SitemapItem {
 
   buildXML () {
     this.url.children = []
-    this.url.attributes = {}
+    this.url.attribs = {}
```

Comment:
XMLbuilder [version 11](https://github.com/oozcitak/xmlbuilder-js/blob/master/CHANGELOG.md#1100---2019-02-18) changed this prop to attribs